sync_diff_inspector/config_sharding.toml

```diff
@@ -0,0 +1,96 @@
+# diff Configuration.
```

Comment:
config_sharding.toml is a example for diff sharding mode table's data.
config.toml is a example for diff table one to one.